libsolidity/AST.h

```diff
@@ -166,6 +166,21 @@ class Declaration: public ASTNode
 };
 
 /**
+ * Declaration of an Enum Value
```

Comment:
Could you move that right above or below EnumDefinition? The classes around here are quite general beasts.